.gitignore

```diff
@@ -60,4 +60,7 @@ ios/Frameworks/
 ios/Runner.xcworkspace/xcshareddata/
 example/ios/Flutter/flutter_export_environment.sh
 flutter_showcaseview.iml
-pubspec.lock
\ No newline at end of file
+pubspec.lock
```

Comment:
Doesn't `example` have a `.gitignore` file too? It should be added there!

Reply:
Moved to example .gitignore